libs/sdk-ui-tests/stories/visual-regression/visualizations/insightStories.tsx

```diff
@@ -270,6 +270,11 @@ ScenarioGroupsByVis.forEach((groups) => {
         const visualOnly: ScenarioGroup<any> = group.forTestTypes("visual");
 
         visualOnly.scenarioList.forEach((scenario) => {
+            if (scenario.tags.indexOf("no-plug-viz-tests") !== -1) {
```

Comment:
Nitpick: can be simplified with `includes`

Reply:
fixed :-)

Reply:
Nice ...